### PR TITLE
Added devstack config yml file to app repo

### PR DESCRIPTION
**Pull request by syedimranhassan**
## Description

TODO: Include a detailed description of the changes in the body of the PR

### CODE REVIEW

**Review by kdmccormick — COMMENTED** [review]
Could you give some more context on the new file @syedimranhassan ? For example, where is the file loaded, and what is it a replacement for?

**Comment by syedimranhassan** [comment]
@kdmccormick working on [this](https://openedx.atlassian.net/browse/OPS-4397) ticket. We are in process of removing rendering code tasks. So, we should hard code file in devstack so it'll not break.

File: registrar/settings/registrar.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+# Ansible managed
```
Comment (kdmccormick) [inline registrar/settings/registrar.yml]: can you remove this comment?

**Review by kdmccormick — COMMENTED** [review]
Thanks for the context. As far as I know, we don't actually use the rendered YAML for devstack -- we just declare all settings manually in https://github.com/edx/registrar/blob/master/registrar/settings/devstack.py . So I'm not sure that this file would actually be used anywhere.

However if you think this should still be merged because it's part of the larger config-simplification plan, go ahead.

File: registrar/settings/registrar.yml (hunk 1 above)
Comment (syedimranhassan) [reply registrar/settings/registrar.yml]: @kdmccormick I have addressed your comment. Can you please review it?

**Review by kdmccormick — APPROVED** [review]
Sorry for the delay. I maintain that this file won't actually do anything currently, but I put a ticket in our backlog to change that: https://openedx.atlassian.net/browse/EDUCATOR-4802

fyi @schenedx

**Pull request merged**